lib/sqlalchemy/orm/scoping.py

```diff
@@ -470,6 +470,10 @@ def begin_nested(self) -> SessionTransaction:
             with the SQLite driver in order for SAVEPOINT to work
             correctly.
 
+            :ref:`aiosqlite_serializable` - special workarounds required
+            with the SQLite driver in order for SAVEPOINT to work
+            correctly.
```

Comment:
**Federico Caselli** (CaselIT) wrote:

can we collapse the two, maybe adding `The same holds true for the aiosqlite driver :ref...`

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4907

Reply:
Fixed